test/e2e/common/downward_api.go

```diff
@@ -90,7 +90,6 @@ var _ = Describe("[sig-node] Downward API", func() {
 	   Description: Downward API MUST expose Pod and Container fields as environment variables. Specify host IP as environment variable in the Pod Spec are visible at runtime in the container.
 	*/
 	framework.ConformanceIt("should provide host IP as an env var [NodeConformance]", func() {
-		framework.SkipUnlessServerVersionGTE(hostIPVersion, f.ClientSet.Discovery())
```

Comment:
Given how old this behavior is I'm ok with the removal, but should the consts hostIPVersion and podUIDVersion also be removed?

Reply:
Oh, nice catch!
I didn't see that.
We should remove them for sure.